### DISCUSSION TITLE
new feathers discussion

### DISCUSSION THREAD

**Discussion by edwardsmarkf** [feathersjs/feathers#2642] [General]
hello - what is the name of the new feathersjs discussion page (not slack obviously)  sorry, i forgot to bookmark it.

specifically, i want to ask a few opinions on this article on typescript:

https://dioxmio.medium.com/typescript-is-overrated-802ab6e3b82b

specifically this statement:

_**There is a good reason JavaScript does not include typing.**_

**Comment by bwgjoseph** [comment]
This is the invite link - https://discord.gg/SGyJzPfk

Can't find the official URL

---

If you didn't read till the end, I think it's a April Fool joke article? See the bottom of the article and there is already a [proposal to introduce typing to javascript](https://devblogs.microsoft.com/typescript/a-proposal-for-type-syntax-in-javascript/)
